### PR TITLE
audio: add enemy death sound effect

### PR DESCRIPTION
**Pull request by ttrssreal**
mrow mrow

### CODE REVIEW

File: scripts/enemy.gd [hunk 1]
```diff
@@ -19,79 +19,80 @@ var speed_reduced = false
 var speed_reduction_timer = null
 
 func _ready() -> void:
-	player = get_node("/root/Main/Player/CharacterBody2D")
-	speed_reduction_timer = Timer.new()
-	speed_reduction_timer.wait_time = speed_reduction_time  # Duration of speed reduction
-	speed_reduction_timer.one_shot = true
-	speed_reduction_timer.connect("timeout", self._on_speed_reduction_timeout)
-	add_child(speed_reduction_timer)
+    player = get_node("/root/Main/Player/CharacterBody2D")
```
Comment (sticks-stuff) [inline scripts/enemy.gd]: LGBT

**Review by sticks-stuff — CHANGES_REQUESTED** [review]
dont fucking change my beautiful children to spaces

**Review by edenstarkey — APPROVED** [review]
meow

**Comment by sticks-stuff** [comment]
hate crime

**Review by sticks-stuff — CHANGES_REQUESTED** [review]
LBTM

**Comment by Izzy-Irvine** [comment]
```
 _._     _,-'""`-._
(,-.`._,'(       |\`-/|
    `-.-' \ )-`( , o o)
          `-    \`_`"'-
```          
My tabs!

**Comment by Izzy-Irvine** [comment]
:shipit:

**Comment by edenstarkey** [comment]
:shipit: :shipit: :shipit:

**Pull request merged**